Makefile

```diff
@@ -6,36 +6,40 @@ XC_ARGS := -project $(PROJECT).xcodeproj -scheme $(SCHEME) -destination $(DESTIN
 XC_BUILD_ARGS := ONLY_ACTIVE_ARCH=NO OTHER_CFLAGS="-fembed-bitcode"
 XC_TEST_ARGS := GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES
 
-bootstrap:
+bootstrap: ## downlaod & install deps
 	./bootstrap.sh
 
-clean:
+clean: ## clean
 	xcodebuild $(XC_ARGS) clean
 
-test:
+test: ## run all tests
 	xcodebuild test $(XC_ARGS)
 
-build:
+build: ## build module
 	xcodebuild $(XC_ARGS) $(XC_BUILD_ARGS)
 
-lint:
+lint: ## lint (all?) code in repo
 	pod lib lint --verbose
 
-format:
+format: ## format (all?) code in repo
```

Comment:
@pingxia0823 can you confirm? I think this formats the code in `RadarSDK/`

Reply:
right this will lint all .h and .m files in repo

Reply:
changed